genesyscloud/resource_genesyscloud_journey_segment.go

```diff
@@ -52,18 +52,10 @@ var (
 				return r
 			}(), ""),
 		},
-		"scope": {
-			Description:  "The target entity that a segment applies to.Valid values: Session, Customer. Changing the scope attribute will cause the existing journey_segment to be dropped and recreated with new ID.",
-			Type:         schema.TypeString,
-			Required:     true,
-			ForceNew:     true, // scope can be only set during creation
-			ValidateFunc: validation.StringInSlice([]string{"Session"}, false),
```

Comment:
As mentioned, in the previous PR the value "Customer" was removed.

The API only allows Session, and the field is not required. Since only session scoped segments exist today, there's no need to declare the scope at all.

Reply:
If you consider this a breaking change because the field was required, let me know and I'll rework this PR to just update the tests / descriptions.

Reply:
I think it would better to add a deprecation message stating that the field is no longer in use by the API, won't do anything, and will be removed in a future version. At least that way we might be giving some context to any users that have implemented it in a system already.